core/vm/interface.go

```diff
@@ -51,11 +51,11 @@ type StateDB interface {
 	GetTransientState(addr common.Address, key common.Hash) common.Hash
 	SetTransientState(addr common.Address, key, value common.Hash)
 
-	Suicide(common.Address) bool
```

Comment:
is the return value removed because it isn't used anywhere?

Reply:
Yes